payments/tests/conftest.py

```diff
@@ -25,7 +26,13 @@
 from .utils import random_price, random_tax
 
 FAKE_BAMBORA_API_URL = "https://fake-bambora-api-url/api"
-UI_RETURN_URL = "https://front-end-url/{LANG}/"
+UI_RETURN_URL = "https://front-end-url/{LANG}"
```

Comment:
do we need this here, if you added it to 
```python
@pytest.fixture(autouse=True)
def force_settings(settings):
    settings.MAILER_EMAIL_BACKEND = "django.core.mail.backends.locmem.EmailBackend"
    settings.DEFAULT_FROM_EMAIL = "noreply@foo.bar"
    settings.LANGUAGE_CODE = "en"
    settings.VENE_UI_RETURN_URL = "https://front-end-url/{LANG}"
```

Reply:
or is this used for testing that the URL gets built properly in different places?

Reply:
It's being used in a couple of places, namely, for the `DummyContext` on the notifications and in some of the tests. We probably could remove it and always get the value from `settings` 🤔

Reply:
easier to keep track of only one var vs two vars :d0rp: